accounts/ui/components/accounts/AccountsList.vue

```diff
@@ -3,13 +3,13 @@
     <oc-table middle divider>
       <oc-table-group>
         <oc-table-row class="fix-table-header">
-          <oc-table-cell shrink type="head">
+          <oc-table-cell shrink type="head" class="uk-text-center">
             <oc-checkbox
-              :value="areAllAccountsSelected"
-              :label="$gettext('Select all users')"
-              hide-label
-              @change="toggleSelectionAll"
-              class="uk-margin-small-left"
+                class="oc-ml-s"
```

Comment:
Some odd change of indentation?

Reply:
True, that is incorrect indentation. We should setup prettier in accounts and settings UI....

I merged the PR with this. I have another PR touching the accounts and settings UI anyway (moving the navItems to the user menu). I will fix the indentation in that PR then. Thanks for spotting it!